calico-vpp-agent/services/service_server.go

```diff
@@ -81,6 +89,64 @@ func (s *Server) SetOurBGPSpec(nodeBGPSpec *common.LocalNodeSpec) {
 	s.nodeBGPSpec = nodeBGPSpec
 }
 
+func (s *Server) ParseServiceAnnotations(annotations map[string]string) *serviceInfo {
+	var err []error
+	svc := &serviceInfo{}
+	for key, value := range annotations {
+		switch key {
+		case LBTypeAnnotation:
```

Comment:
We should prefix annotations with `VppAnnotationPrefix` for consistency with pod